ng/src/web/pages/notes/component.js

```diff
@@ -23,27 +24,35 @@
 
 import React from 'react';
 
-import _ from 'gmp/locale.js';
-import {is_defined, is_empty, shorten} from 'gmp/utils';
+import _ from 'gmp/locale';
+import {
+  is_array,
+  is_defined,
+  is_empty,
+  shorten,
+} from 'gmp/utils';
 
-import PropTypes from '../../utils/proptypes.js';
+import PropTypes from '../../utils/proptypes';
 
-import Wrapper from '../../components/layout/wrapper.js';
+import Wrapper from '../../components/layout/wrapper';
 
-import EntityComponent from '../../entity/component.js';
+import EntityComponent from '../../entity/component';
 
-import NoteDialog from './dialog.js';
+import NoteDialog from './dialog';
 
 class NoteComponent extends React.Component {
 
   constructor(...args) {
     super(...args);
 
+    this.state = {dialogVisible: false};
+
+    this.closeNoteDialog = this.closeNoteDialog.bind(this);
     this.openNoteDialog = this.openNoteDialog.bind(this);
     this.openCreateNoteDialog = this.openCreateNoteDialog.bind(this);
   }
 
-  openNoteDialog(note) {
+  openNoteDialog(note, initial) {
     if (is_defined(note)) {
       let active = '0';
```

Comment:
Would be nice to use come constants for the possible values of active. It would allow easier refactoring if something has to change in future.